packages/web-components/package.json

```diff
@@ -74,6 +74,7 @@
     "karma-source-map-support": "^1.4.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^4.0.2",
+    "lodash-es": "^4.17.20",
```

Comment:
Technically we need to add a peer dependency along with this as well (though I can't suggest it for some reason). 

Can you add:
```
"peerDependencies": {
    "lodash-es": "^4.0.0"
  },
  ```

Reply:
I don't think so; I don't think it is @fluentui/web-components that requires lodash; it's the fast components package that requires the peer dep. 

Which technically means anything which depends on it to build needs to resolve lodash-es. So, if a consumer depends on @fluentui/react, and it depends on packages X Y and Z which each have their own peer deps, the endpoint will need to resolve all of those.

I have the change marked as none.

Reply:
Gotcha - thanks for the update there. I see it's coming from the color work which is in a different package here.